### PR TITLE
doc: clarify INSTALL_LOCK behavior

### PR DESCRIPTION
**Pull request by rkaiser0324**
The previous language was not as clear.  I think this conveys the intent, based on reviewing the code. However I note that INSTALL_LOCK also looks used in other places (e.g., `dump.go`) so probably it's impact elsewhere should also be noted here.

### CODE REVIEW

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md [hunk 1]
```diff
@@ -477,7 +477,7 @@ Certain queues have defaults that override the defaults set in `[queue]` (this o
 
 ## Security (`security`)
 
-- `INSTALL_LOCK`: **false**: Disallow access to the install page.
+- `INSTALL_LOCK`: **false**: Controls access to the install page.  When set to "true", the page is not accessible.
```
Comment (lunny) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md]:
```suggestion
- `INSTALL_LOCK`: **false**: Controls access to the install page. When set to "true", the installation page is not accessible.
```

Comment (wxiaoguang) [reply docs/content/doc/advanced/config-cheat-sheet.en-us.md]: There is another `install page` before 😂

Should we call it `install page` or `installation page`? I have no idea ....

Comment (techknowlogick) [reply docs/content/doc/advanced/config-cheat-sheet.en-us.md]: I agree, I think installation page is better.

Comment (KN4CK3R) [reply docs/content/doc/advanced/config-cheat-sheet.en-us.md]: And there are multiple spaces infront of `When`.

**Review by wxiaoguang — APPROVED** [review]

**Comment by wxiaoguang** [comment]
I fixed these texts, need another LGTM.

**Review by lunny — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**